Sources/StreamChat/Database/DTOs/MemberModelDTO.swift

```diff
@@ -19,9 +19,9 @@ class MemberDTO: NSManagedObject {
     @NSManaged var isBanned: Bool
     @NSManaged var isShadowBanned: Bool
     
-    //  @NSManaged var invitedAcceptedAt: Date?
-    //  @NSManaged var invitedRejectedAt: Date?
-    //  @NSManaged var isInvited: Bool
+    @NSManaged var inviteAcceptedAt: Date?
+    @NSManaged var inviteRejectedAt: Date?
+    @NSManaged var isInvited: Bool
```

Comment:
do these fields exist in core data model?

Reply:
Yes, they do